bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -424,4 +443,35 @@ public AutoSaveConfiguration getAutoSaveConfig() {
         return autoSaveConfig;
     }
 
+    @Override
+    public Map<String, Object> getSubmitProperties() {
+        Map<String, Object> submitProps = new LinkedHashMap<>();
+        Map<String, String> submitActionMap = new LinkedHashMap<>();
+
+        submitActionMap.put(FormConstants.RT_SUBMIT_TYPE_EMAIL, FormConstants.NAME_SUBMIT_TYPE_EMAIL);
```

Comment:
use the existing GuideConstants.DEFAULT_GUIDE_SUBMIT_ACTION_SERVICES

Reply:
DONE, Created properties in cc for now as forms-sdk release can't be done right now. Had this discussion on slack